test/blocklib/algorithm/fft/fft_types.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef GRAPH_PROTOTYPE_ALGORITHM_FFT_TYPES_HPP
+#define GRAPH_PROTOTYPE_ALGORITHM_FFT_TYPES_HPP
+
+#include "node.hpp"
```

Comment:
IMO this is better inlined where this is specifically needed to keep the logic and reasoning of the code local.

Reply:
This types are used at least for both FFt algorithms and in FFT QA. Probably it can be simplified if we do not support some rare combinations of input and output but just complex numbers, floats and doubles.